cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MessagesApiModelExt.kt

```diff
@@ -8,19 +9,44 @@ import com.sourcepoint.cmplibrary.model.CampaignReq
 import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.* // ktlint-disable
 
-internal fun getMessageBody(
-    propertyHref: String,
+private const val GET_MESSAGES_REQ_BODY_ACCOUNT_ID_KEY = "accountId"
+private const val GET_MESSAGES_REQ_BODY_PROPERTY_HREF_KEY = "propertyHref"
+private const val GET_MESSAGES_REQ_BODY_CAMPAIGNS_KEY = "campaigns"
+private const val GET_MESSAGES_REQ_BODY_CAMPAIGN_ENV_KEY = "campaignEnv"
+private const val GET_MESSAGES_REQ_BODY_CONSENT_LANGUAGE_KEY = "consentLanguage"
+private const val GET_MESSAGES_REQ_BODY_HAS_CSP_KEY = "hasCSP"
+private const val GET_MESSAGES_REQ_INCLUDE_DATA_KEY = "includeData"
+private const val GET_MESSAGES_REQ_LOCAL_STATE_KEY = "localState"
+private const val GET_MESSAGES_REQ_OS_KEY = "os"
+
+internal fun createGetMessagesRequestBody(
```

Comment:
@andresilveirah actually, we can do that, and that will be better in terms of unifying the way we deal with our network request, but that will require some time, since I have to find a better way to deal with `IncludeData` param.

We use `IncludeData` params in a bunch of our requests (messages, consent, get choice, etc), and we pretty much deal with those in a different way, so it is all over the place.